components/shared/BorderCard.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <v-card
+    :style="style"
+    class="pa-10 card-border"
+  >
+    <slot />
+  </v-card>
+</template>
+
+<script>
+export default {
+  props: ['color'],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rafaelmbcosta/league-nuxt/pullRequest?prid=7487498)
```suggestion
  props: ["color"],
```